guide/14-deep-learning/inference-only-text-models-in-arcgis-learn.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<p>Some explanation /one liner for this call should be explained as it is different for other models in the sense that we don not need to pass in data </p><p> </p><p>This cell created a model object by calling the <code>ZeroShotClassifier </code>class. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Made the necessary changes</p>